docs/src/developing/clients/jsonrpc-api.md

```diff
@@ -2125,21 +2125,20 @@ Result:
 
 ### getRecentPrioritizationFees
 
-**UNSTABLE**
-Returns a list of minimum prioritization fees from recent blocks. Currently, a
+Returns a list of prioritization fees from recent blocks. Currently, a
 node's prioritization-fee cache stores data from up to 150 blocks.
 
 #### Parameters:
 
-- `<array>` - (optional) An array of account address strings (up to a maximum of 128 addresses). If this parameter is provided, the response will reflect the minimum prioritization fee to land a transaction locking all of the provided accounts as writable.
+- `<array>` - (optional) An array of account address strings (up to a maximum of 128 addresses). If this parameter is provided, the response will reflect a fee to land a transaction locking all of the provided accounts as writable.
```

Comment:
The naked array input here is actually a little unfortunate too. One day read-only accounts (or even executables) may affect prioritization as well. I guess the input becomes an array *or* an object

Reply:
Yeah, that's true. Sigh, I should know better by now, and just use an input config object for any and all new endpoints.
Fortunately, I have lots of practice with the enum shenanigans to fix.